crates/containerd-shim-wasmer/Cargo.toml

```diff
@@ -11,6 +11,7 @@ log = { workspace = true }
 tokio = "1.38.1"
 wasmer = { version = "4.1.2" }
 wasmer-wasix = { version = "0.12.0" }
+mio = { version = "0.8.11", features = ["net"] }
```

Comment:
**question:** why do we need this new dep?

Reply:
To resolve this issue: https://github.com/containerd/runwasi/actions/runs/11224084236/job/31200120860#step:6:400